### PR TITLE
Phase2-sim73 Add provision different regions for fine detectors

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Add provision different regions for fine detectors

#### PR validation:

Useful for studies of HGCal reconstruction studies

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

Nothing special

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by bsunanda** [comment]
@cmsbuild Please test

File: Geometry/HcalSimData/data/CaloUtil/2026/v0/CaloUtil.xml [hunk 1]
```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0"?>
```
Comment (cvuosalo) [inline Geometry/HcalSimData/data/CaloUtil/2026/v0/CaloUtil.xml:1]: What does "v0" mean? We usually start with "v1".

Comment (bsunanda) [reply Geometry/HcalSimData/data/CaloUtil/2026/v0/CaloUtil.xml:1]: OK I shall do that

**Comment by cvuosalo** [comment]
+1

**Comment by qliphy** [comment]
+1

**Pull request merged**